packages/react-components/react-menu/src/stories/MenuControllingOpenAndClose.stories.tsx

```diff
@@ -3,16 +3,18 @@ import * as React from 'react';
 import { Menu, MenuTrigger, MenuList, MenuItem, MenuPopover, MenuProps } from '../index';
 
 import { Button } from '@fluentui/react-button';
-import { Checkbox } from '@fluentui/react-checkbox';
+import { Checkbox, CheckboxProps } from '@fluentui/react-checkbox';
 
 export const ControllingOpenAndClose = () => {
   const [open, setOpen] = React.useState(false);
   const onOpenChange: MenuProps['onOpenChange'] = (e, data) => {
     setOpen(data.open);
   };
 
-  const onChange = (e: React.ChangeEvent<HTMLInputElement>) => {
-    setOpen(e.target.checked);
+  const onChange: CheckboxProps['onChange'] = (e: React.ChangeEvent<HTMLInputElement>, { checked }) => {
```

Comment:
```suggestion
  const onChange: CheckboxProps['onChange'] = (e, { checked }) => {
```

you don't need explicit typing anymore